packages/kbn-test/src/functional_tests/run_tests/flags.ts

```diff
@@ -50,6 +52,13 @@ export const FLAG_OPTIONS: FlagOptions = {
     --updateSnapshots    Replace inline and file snapshots with whatever is generated from the test
     --updateAll, -u      Replace both baseline screenshots and snapshots
   `,
+  examples: `
```

Comment:
The arg parsing/help printing in these runners is a bit too convoluted, but this commit shows the examples: [02c089c](https://github.com/elastic/kibana/pull/166054/commits/02c089ca911159643cdffe561010c5b99179ff6e)

I'm not sure if it's worth the wiring of the examples to show this

Reply:
I think it is good enough